README.md

```diff
@@ -167,7 +167,7 @@ If the module doesn't exist on `nf-core/modules`:
 We have implemented a number of commands in the `nf-core/tools` package to make it incredibly easy for you to create and contribute your own modules to nf-core/modules.
 
 1. Install the latest version of [`nf-core/tools`](https://github.com/nf-core/tools#installation) (`>=1.13`)
-2. Install [`nextflow`](https://nf-co.re/usage/installation) (`>=20.11.0-edge`; see [Nextflow edge releases](#nextflow-edge-releases))
+2. Install [`nextflow`](https://nf-co.re/usage/installation) (`>=21.04.0`; see [Nextflow edge releases](#nextflow-edge-releases))
```

Comment:
```suggestion
2. Install [`Nextflow`](https://nf-co.re/usage/installation) (`>=21.04.0`; see [installation docs](https://www.nextflow.io/docs/latest/getstarted.html#installation))
```

Reply:
We can remove all of the edge release docs here now and replace it with a link to main NF installation docs as I have done above. I kick-started them being added to the main NF docs so we can just link to those everywhere now.